infrastructure/template.yaml

```diff
@@ -117,7 +117,7 @@ Resources:
   OTGCanaries:
     Type: AWS::Synthetics::Canary
     Properties:
-      Name: get-bearer-token
+      Name: !Sub ${AWS::StackName}-ci
```

Comment:
This might cause a problem when you have long stack names. Canary has a name length limit afaik